ethereum/spec/src/main/java/tech/pegasys/teku/spec/networks/Eth2Network.java

```diff
@@ -19,7 +19,6 @@
 public enum Eth2Network {
   // Live networks
   MAINNET,
-  PYRMONT,
```

Comment:
Also need to remove [line 22](https://github.com/ConsenSys/teku/pull/4405/files#R22) of _Eth2NetworkConfiguration.java_ else compilation fails:

```
import static tech.pegasys.teku.spec.networks.Eth2Network.PYRMONT;
```

Reply:
Running `./gradlew spotlessApply` will catch errors like this and also ensure that the code formatting conforms to our specification (building will fail if it doesn't).

Running `./gradlew build` locally compiles the code and runs the unit tests. This is well worth doing to catch issues before committing.